client/src/routerPaths.ts

```diff
@@ -20,10 +20,10 @@ const routerPaths = {
   }: BallotScreenProps) =>
     `/ballots/style/${ballotStyleId}/precinct/${precinctId}/language/${localeCode}`,
   printedBallotsReport: '/ballots/printed-report',
-  printTestDecks: '/ballots/print-test-deck',
-  printOneTestDeck: ({ precinctId }: PrecinctReportScreenProps) =>
-    `/ballots/print-test-deck/${precinctId}`,
   tally: '/tally',
+  printTestDecks: '/tally/print-test-deck',
+  printOneTestDeck: ({ precinctId }: PrecinctReportScreenProps) =>
+    `/tally/print-test-deck/${precinctId}`,
```

Comment:
I'm concerned about these scope creep nav changes so close to code freeze.

Reply:
just to be clear about this change: it was buggy before. You'd go to "tally" then click on "print test deck", and the tab would switch to "ballots".

Reply:
…because the user is printing "ballots". We had never talked about this and I just don't want anything to break unintentionally. 👍

Reply:
Yeah, I guess then the button should probably have been under ballots to begin with?

You're right to point out that this change was stealth even if it ws not meant to be.

Thanks for highlighting and the approval nevertheless. I'll take it upon myself to test this well.